.github/actions/auto-merge.yml

```diff
@@ -0,0 +1,15 @@
+name: auto-merge
+
+on:
+  pull_request:
+    paths-ignore:
+      - "src/**" # prevent auto-merge for go dependencies
```

Comment:
Unfortunately auto-merge [config](https://github.com/marketplace/actions/dependabot-auto-merge#configuration-file-syntax) does not support [`ignore` options](https://github.com/ahmadnassri/action-dependabot-auto-merge/pull/43). 

[paths-ignore](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-excluding-paths) prevents the auto-merge action from running for `go` dependencies under `src/`. 

![image](https://user-images.githubusercontent.com/7441475/198755502-9ea7c6a6-6ca3-4d1c-9df1-ee48960f1576.png)